go.mod

```diff
@@ -41,9 +41,12 @@ require (
 	schneider.vip/problem v1.6.0
 )
 
+require github.com/ethereum/go-ethereum v1.10.13
```

Comment:
wait, what?

Reply:
Surprise! In all seriousness, I'll try to replace it with another library. If that fails we're going to create a repo with just the `crypto/ecies` package of Ethereum.